pkg/virtual/workspaces/registry/rest.go

```diff
@@ -326,6 +327,18 @@ func (s *REST) Create(ctx context.Context, obj runtime.Object, createValidation
 			Type: workspace.Spec.Type,
 		},
 	}
+
+	ownerRaw, err := clusterworkspaceadmission.ClusterWorkspaceOwnerAnnotationValue(userInfo)
+	if err != nil {
+		return nil, fmt.Errorf("error constructing workspace owner annotation from user info: %w", err)
+	}
+
+	if clusterWorkspace.Annotations == nil {
+		clusterWorkspace.Annotations = make(map[string]string)
+	}
+
+	clusterWorkspace.Annotations[tenancyv1alpha1.ExperimentalClusterWorkspaceOwnerAnnotationKey] = ownerRaw
+
```

Comment:
this confuses me. How did we set this before?

Reply:
it was missing in this flow